ush/config_defaults.yaml

```diff
@@ -2316,92 +2412,28 @@ global:
-  SKEB_INT: 3600 #Variable "skebint" in input.nml
-  SKEBNORM: 1
-  SKEB_VDOF: 10
-  USE_ZMTNBLCK: false
-  #
-  #-----------------------------------------------------------------------
-  #
-  # Set default SPP stochastic physics options. 
-  # SPP perturbs specific tuning parameters within a physics parameterization 
-  # (unlike SPPT, which multiplies overall physics tendencies by a random  
-  # perturbation field *after* the call to the physics suite).Patterns evolve 
-  # and are applied at each time step. Each SPP option is an array, 
-  # applicable (in order) to the HRRR-based parameterization listed in SPP_VAR_LIST. 
-  # Enter each value of the array in config.yaml as shown below without commas
-  # or single quotes (e.g., SPP_VAR_LIST: [ "pbl" "sfc" "mp" "rad" "gwd" ] ). 
-  # Both commas and single quotes will be added by Jinja when creating the
-  # namelist.
-  #
-  # Note that SPP is currently only available for specific physics schemes 
-  # used in the RAP/HRRR physics suite. Users need to be aware of which SDF
-  # is chosen when turning this option on. 
-  #
-  #-----------------------------------------------------------------------
-  #
-  DO_SPP: false
-  SPP_VAR_LIST: [ "pbl", "sfc", "mp", "rad", "gwd" ]
-  SPP_MAG_LIST: [ 0.2, 0.2, 0.75, 0.2, 0.2 ] #Variable "spp_prt_list" in input.nml
-  SPP_LSCALE: [ 150000.0, 150000.0, 150000.0, 150000.0, 150000.0 ]
-  SPP_TSCALE: [ 21600.0, 21600.0, 21600.0, 21600.0, 21600.0 ] #Variable "spp_tau" in input.nml
-  SPP_SIGTOP1: [ 0.1, 0.1, 0.1, 0.1, 0.1 ]
-  SPP_SIGTOP2: [ 0.025, 0.025, 0.025, 0.025, 0.025 ]
-  SPP_STDDEV_CUTOFF: [ 1.5, 1.5, 2.5, 1.5, 1.5 ]
-  ISEED_SPP: [ 4, 5, 6, 7, 8 ]
   #
   #-----------------------------------------------------------------------
   #
   # Turn on SPP in Noah or RUC LSM (support for Noah MP is in progress).
   # Please be aware of the SDF that you choose if you wish to turn on LSM
   # SPP.
   #
-  # SPP in LSM schemes is handled in the &nam_sfcperts namelist block 
-  # instead of in &nam_sppperts, where all other SPP is implemented.
-  #
   # Perturbations to soil moisture content (SMC) are only applied at the 
   # first time step.
   #
-  # LSM perturbations include SMC - soil moisture content (volume fraction),
-  # VGF - vegetation fraction, ALB - albedo, SAL - salinity, 
-  # EMI - emissivity, ZOL - surface roughness (cm), and STC - soil temperature.
-  #
-  # Only five perturbations at a time can be applied currently, but all seven
-  # are shown below.  In addition, only one unique iseed value is allowed 
-  # at the moment, and is used for each pattern.
-  #
-  DO_LSM_SPP: false #If true, sets lndp_type=2
-  LSM_SPP_TSCALE: [ 21600, 21600, 21600, 21600, 21600, 21600, 21600 ]
-  LSM_SPP_LSCALE: [ 150000, 150000, 150000, 150000, 150000, 150000, 150000 ]
-  ISEED_LSM_SPP: [ 9 ]
-  LSM_SPP_VAR_LIST: [ "smc", "vgf", "alb", "sal", "emi", "zol", "stc" ]
-  LSM_SPP_MAG_LIST: [ 0.017, 0.001, 0.001, 0.001, 0.001, 0.001, 0.2 ]
+  DO_LSM_SPP: false
```

Comment:
This also needed to be added back in. I set the value as `false` to mimic what the value was previously.